packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [8.0.1](https://github.com/parse-community/Parse-SDK-Flutter/compare/flutter-8.0.0...flutter-8.0.1) (2024-11-28)
```

Comment:
If this is a feature PR, then the version should be 8.1.0

Reply:
added.